src/todolist/todolist.service.ts

```diff
@@ -17,21 +16,19 @@ import {
 import {
   CreateTodolistItemRequest,
   CreateTodolistRequest,
-  ItemDTO,
-  TodolistDTO,
   UpdateTodolistItemRequest,
   UpdateTodolistRequest,
 } from './dto/TodolistDTO';
 import { Todolist } from '../entities/Todolist';
-import { TodolistItem } from '../entities/TodoItem';
 import { mapItemFromEntity, mapTodolistFromEntity } from './utils';
+import { SharingServiceLocalClient } from '../sharing/service/SharingServiceClient';
 
 @Injectable()
 export class TodolistService {
   constructor(
     private readonly repository: TodolistRepository,
     private readonly userService: UserService,
-    private readonly shareService: SharingService,
+    private readonly shareService: SharingServiceLocalClient,
```

Comment:
Changed to client class but we could also make it further generalised to the interface with a very small change. Just demonstrative here.